tests/routes/datasets/test_versions.py

```diff
@@ -407,16 +409,13 @@ async def test_put_latest(async_client):
     )
 
 
-@pytest.mark.hanging
 @pytest.mark.asyncio
 async def test_version_put_raster(async_client):
     """Test raster source version operations."""
 
     dataset = "test_version_put_raster"
     version = "v1.0.0"
 
-    s3_client = get_s3_client()
-
```

Comment:
Just to understand, what is the motivation to move this outside the test?

Reply:
Hmm, I can't remember. I'm guessing it's left over from some early refactoring that I didn't finish. Will remove, sorry about the noise.